KukaApp/__tests__/BootComponent.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import * as eva from '@eva-design/eva';
+import { SafeAreaProvider } from 'react-native-safe-area-context';
+import { ApplicationProvider, IconRegistry } from '@ui-kitten/components';
+import { EvaIconsPack } from '@ui-kitten/eva-icons';
+import { default as theme } from '../app-theme.json';
+import { default as mapping } from '../app-mapping.json';
+
+export const BootComponent = props => {
+  return (
+    <>
+      <IconRegistry icons={EvaIconsPack} />
+      <ApplicationProvider
+        {...eva}
+        theme={{ ...eva.light, ...theme }}
+        customMapping={mapping}
+      >
+        <SafeAreaProvider
```

Comment:
Components are referencing the "safe area" as such we'll need to make this a standard. This benefits us too if we use this for testing the screens. Granted I usually do a `BootScreen` and `BootComponent` as those each should have a good amount of separation of concerns.